EIPS/eip-1901.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: 1901
+title: Add OpenRPC Service Discovery To JSON-RPC Services
+author: Shane Jonas (@shanejonas), Zachary Belford (@belfordz)
+discussions-to: https://github.com/ethereum/EIPs/issues/1902
+status: Draft
+type: Standards Track
+category: Interface
+created: 2019-02-25
+---
+
+## Abstract
+### What is this?
+
+This is a proposal to add [OpenRPC](https://github.com/open-rpc/spec) support to existing and future JSON-RPC services by adding the method [`rpc.discover`](https://github.com/open-rpc/spec#service-discovery-method) to the projects [JSON-RPC](https://www.jsonrpc.org/specification) APIs, enabling automation and tooling.
+
+The OpenRPC Document and generated Documentation that specifies all the methods an EVM-based blockchain should implement can be found [here](https://github.com/etclabscore/ethereum-json-rpc-specification).
+
+This was first proposed [here as an ECIP](https://github.com/etclabscore/ECIPs/blob/master/ECIPs/ECIP-1053.md), but the benefits of this kind of tooling is apparent across Bitcoin, Ethereum Classic, Ethereum and other JSON-RPC accessible blockchains.
+
+## Motivation
+EVM-based blockchains lack a form RPC Specification. This proposal attempts to standardize such a specification in a way that is versionable, and both human and machine readable.
```

Comment:
I think this should just say "Ethereum" since this is the EIPs repo. Also note there is a spec: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md